prow/jobs/control-plane/components/schema-migrator/schema-migrator-kcp-generic.yaml

```diff
@@ -5,7 +5,7 @@ presubmits: # runs on PRs
   kyma-project/control-plane:
     - name: pre-control-plane-components-schema-migrator
       annotations:
-        owner: "gophers, framefrog"
```

Comment:
In some places there is no space but in others there is. Is that intentional?

Reply:
Nope, but it shouldn't matter anyway.

Reply:
Could you please remove the space? Thank you in advance.